PCX-10263 Clarify requirements for purpose justification approvers (#13583)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/access/temporary-auth.md

3. Choose the **Allow** policy you want to configure and select **Edit**.
4. Under **Additional settings**, turn on [**Purpose justification**](/cloudflare-one/policies/access/require-purpose-justification/).
5. Turn on **Temporary authentication**.

> 6. Enter the **Email addresses of the approvers**. (Note: your approvers must be authenticated by Access in order to verify their identity. If they do not have an active session it will default to your [Application Launcher Access policy](/cloudflare-one/applications/app-launcher/) to verify their identity.)

Reviewer: ```suggestion
6. Enter the **Email addresses of the approvers**. (Note: your approvers must be authenticated by Access. If they do not have an active session, Access will verify their identity against your [App Launcher policies](/cloudflare-one/applications/app-launcher/).)
```
